sockeye/inference.py

```diff
@@ -183,13 +197,18 @@ def chunks(self, chunk_size: int) -> Generator['TranslatorInput', None, None]:
             # Constrained decoding is not supported for chunked TranslatorInputs. As a fall-back, constraints are
             # assigned to the first chunk
             constraints = self.constraints if chunk_id == 0 else None
+            # Target_prefix_tokens are also assigned only to the first chunk
+            target_prefix_tokens = self.target_prefix_tokens if chunk_id == 0 else None
+            keep_target_prefix_key = self.keep_target_prefix_key if chunk_id == 0 else True
```

Comment:
Good call on stating this explicitly!

This looks like something else we want to configure in the JSON input.  For example, if we're continuing a partial translation, we only want to apply the prefix to the first chunk.  If we're using special target tokens for multilingual NMT, we need to add them to all chunks so we know which language we're translating into.

Reply:
Done, thanks!